docs/templates/legacy-template.md

```diff
@@ -0,0 +1,13 @@
+# LegacyTemplate
+
+[Source file](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/src/BlockTypes/LegacyTemplate.php)
+
+The LegacyTemplate is a class used to set up the Legacy Template block on the server-side, and render the correct template.
```

Comment:
```suggestion
The `LegacyTemplate` is a class used to set up the Legacy Template block on the server-side, and render the correct template.
```

Here again we might want to use the Classic Template name. This is a good point to talk about whether renaming the file and the class itself is safe as discussed [here](https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/5861#issuecomment-1049627578)

Reply:
Ive renamed this to `ClassicTemplate` and even updated the source file link to `ClassicTemplate` in anticipation that is what the file will also be called.